rosidl_parser/rosidl_parser/grammar.lark

```diff
@@ -0,0 +1,518 @@
+// Only implements the following parts of the spec:
+// 7.2.2 Comments
+// 7.2.3 Identifiers
+// 7.2.6 Literals
+// 7.3 Preprocessing
+//   only #include is supported
+// 7.4.1 Building Block Core Data Types
+// 7.4.15.4.2 Applying Annotations
+//   Only supported on modules, structs, members, enums and enum identifiers
+
+%import common.DIGIT
+%import common.HEXDIGIT
+%import common.LETTER
+%import common.WS
+%ignore WS
+
+
+// 7.2.2 Comments
+COMMENT: "//" /[^\n]/*
+  | "/*" /(.|\n)+/ "*/"
+%ignore COMMENT
```

Comment:
Is ignoring comments intentional? I would have expected them to be kept after #323

Reply:
Yes, ignoring comments is intentional. Comments in `.idl` are not extracted. The comments from `.msg` files are bring transformed into annotations in IDL and those are being extracted.